src/sentry/api/endpoints/user_authenticator_enroll.py

```diff
@@ -95,6 +97,12 @@ def get_serializer_field_metadata(serializer, fields=None):
 
 
 class UserAuthenticatorEnrollEndpoint(UserEndpoint):
+    def _check_webauthn_register_ff(self, user):
```

Comment:
it is weird to define this function again, could you move the one in the previous PR to outside of a class and import it here?